mgmtsystem/azure.py

```diff
@@ -73,7 +77,7 @@ def run_script(self, script):
         self.logger.info("PowerShell Returned:\n{}\n".format(result.std_out.strip()))
         return result.std_out.strip()
 
-    def start_vm(self, vm_name, resource_group):
+    def start_vm(self, vm_name, resource_group=default_resource_group):
```

Comment:
This won't work, you should do it like this instead,

```python
def start_vm(self, vm_name, resource_group=None):
    if not resource_group:
        resource_group = self.resource_group
    ....
```

We can also then remove line 52